drivers/hmis/app/models/hmis/hud/custom_client_name.rb

```diff
@@ -46,12 +41,14 @@ class CannotDestroyPrimaryNameException < StandardError
     joins(:client).merge(Hmis::Hud::Client.viewable_by(user))
   end
 
-  def ==(other)
-    columns = [:first, :last, :middle, :suffix, :use]
```

Comment:
@yakloinsteak can you confirm that it's OK for me to remove this function you added? From what I could tell, its not actually being used. I'm looking at MergeClientsJob and it has its own logic for merging names. 

It was messing things up in weird internal ways that I don't really understand.

Reply:
nevermind - I see it broke the spec. I will replace with a function with its own name, like `equal_for_merge?`

Reply:
I can go back and refactor the code to have a custom equality method like you're talking about. Just let me know, and I can prioritize it.

Reply:
It's all good, I replaced them with a function as part of this PR. Nothing fancy but I think my implementation is fine, and tests are happy. thanks!